kubelet-to-gcm/Dockerfile

```diff
@@ -12,15 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM ubuntu:14.04
+FROM scratch
 MAINTAINER Quintin Lee "qlee@google.com"
```

Comment:
Mind removing this? We're removing `MAINTAINER`s from Dockerfiles, since they're deprecated and tend to get out of date.